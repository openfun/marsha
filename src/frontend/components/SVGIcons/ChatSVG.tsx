import React from 'react';

import SVGIcon, { SvgProps } from '.';

export const ChatSVG = (svgProps: SvgProps) => {
  return (
    <SVGIcon viewBox={{ x: 0, y: 0, width: 54, height: 54 }} {...svgProps}>
      <g fillRule="evenodd" stroke="none" strokeWidth="1">
        <g transform="translate(1.5 2.5)">
          <path
            fillRule="nonzero"
            d="M40.208 8.333H11.875c-1.948 0-3.524 1.594-3.524 3.542L8.333 43.75l7.084-7.083h24.791a3.552 3.552 0 003.542-3.542v-21.25a3.552 3.552 0 00-3.542-3.542zm-21.25 21.25h-3.541v-3.541h3.541v3.541zm0-5.312h-3.541v-3.542h3.541v3.542zm0-5.313h-3.541v-3.541h3.541v3.541zm10.625 10.625h-5.312c-.974 0-1.771-.797-1.771-1.77 0-.974.797-1.771 1.77-1.771h5.313c.974 0 1.771.797 1.771 1.77 0 .974-.797 1.771-1.77 1.771zm5.313-5.312H24.27c-.974 0-1.771-.797-1.771-1.771 0-.974.797-1.77 1.77-1.77h10.626c.974 0 1.77.796 1.77 1.77s-.796 1.77-1.77 1.77zm0-5.313H24.27c-.974 0-1.771-.797-1.771-1.77 0-.974.797-1.771 1.77-1.771h10.626c.974 0 1.77.797 1.77 1.77 0 .974-.796 1.771-1.77 1.771z"
          ></path>
        </g>
      </g>
    </SVGIcon>
  );
};
