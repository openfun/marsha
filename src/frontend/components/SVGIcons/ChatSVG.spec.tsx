import React from 'react';
import renderer from 'react-test-renderer';
import { ChatSVG } from './ChatSVG';

it('renders ChatSVG correctly', () => {
  const ChatSVGSnapshot = renderer
    .create(<ChatSVG iconColor={'#035ccd'} />)
    .toJSON();
  expect(ChatSVGSnapshot).toMatchSnapshot();
});
