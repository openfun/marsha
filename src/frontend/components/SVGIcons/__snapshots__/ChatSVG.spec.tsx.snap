// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ChatSVG correctly 1`] = `
.c0 path {
  fill: #035ccd;
}

<svg
  className="c0"
  viewBox="0 0 54 54"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      transform="translate(2.5 2.5)"
    >
      <path
        d="M40.208 8.333H11.875c-1.948 0-3.524 1.594-3.524 3.542L8.333 43.75l7.084-7.083h24.791a3.552 3.552 0 003.542-3.542v-21.25a3.552 3.552 0 00-3.542-3.542zm-21.25 21.25h-3.541v-3.541h3.541v3.541zm0-5.312h-3.541v-3.542h3.541v3.542zm0-5.313h-3.541v-3.541h3.541v3.541zm10.625 10.625h-5.312c-.974 0-1.771-.797-1.771-1.77 0-.974.797-1.771 1.77-1.771h5.313c.974 0 1.771.797 1.771 1.77 0 .974-.797 1.771-1.77 1.771zm5.313-5.312H24.27c-.974 0-1.771-.797-1.771-1.771 0-.974.797-1.77 1.77-1.77h10.626c.974 0 1.77.796 1.77 1.77s-.796 1.77-1.77 1.77zm0-5.313H24.27c-.974 0-1.771-.797-1.771-1.77 0-.974.797-1.771 1.77-1.771h10.626c.974 0 1.77.797 1.77 1.77 0 .974-.796 1.771-1.77 1.771z"
        fillRule="nonzero"
      />
    </g>
  </g>
</svg>
`;
